doc/source/multivariate.rst

```diff
@@ -144,12 +144,12 @@ We realize that the mean vector and the covariance often have special forms in p
 
 .. code-block:: julia
 
-    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector{Float64},ZeroVector{Float64}}} <: AbstractMvNormal
+    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector,ZeroVector}} <: AbstractMvNormal
```

Comment:
I don't have a strong opinion, but the case I had in mind was one where you could save memory if you knew you only needed derivatives with respect to μ, say, and Σ was constant. If both get promoted to the same element type, it's full gradients across the board.

This may be too much worry for a case nobody cares about, though. I'm happy to simply add the type parameter.